src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesSequences.js

```diff
@@ -27,20 +28,18 @@ class SeriesSequences extends Component {
 
   render() {
     const { relations } = this.props;
-
     const sequenceRelations = _get(relations, 'sequence', []);
-    const hasSequenceRelations = sequenceRelations.length > 0;
+    if (_isEmpty(sequenceRelations)) return null;
```

Comment:
I still prefer to use native JS instead of abusing of `isEmpty`, do you really need it here?

Reply:
this one slipped, thanks, fixed.